dotcom-rendering/src/web/components/FixedSmallSlowIV.tsx

```diff
@@ -4,63 +4,72 @@ import { LI } from './Card/components/LI';
 import { UL } from './Card/components/UL';
 
 type Props = {
+	collectionId: string;
 	trails: TrailType[];
 	containerPalette?: DCRContainerPalette;
 	showAge?: boolean;
+	hasMore: boolean;
```

Comment:
What do you think about this property being optional?

Reply:
resolved by consolidation into a single optional prop here: https://github.com/guardian/dotcom-rendering/pull/5463